spec/javascripts/controllers/ems_common/ems_common_form_controller_spec.js

```diff
@@ -63,8 +63,12 @@ describe('emsCommonFormController', function() {
       expect($scope.emsCommonModel.openstack_infra_providers_exist).toEqual(false);
     });
 
-    it('sets the api_port to 5000', function() {
-      expect($scope.emsCommonModel.api_port).toEqual(5000);
+    it('sets the default_api_port to 5000', function() {
+      expect($scope.emsCommonModel.default_api_port).toEqual(5000);
+    });
+
+    it('sets the amqp_api_port to 5000', function() {
+      expect($scope.emsCommonModel.amqp_api_port).toEqual(5000);
```

Comment:
probably should add a test for setting the different IP addresses for the endpoints